autoPlay.user.js

```diff
@@ -926,6 +927,13 @@ function useMetalDetectorIfRelevant() {
 	}
 }
 
+function useDecreaseCooldown(beforeAbility) {
+	if (!isAbilityCoolingDown(ABILITIES.COOLDOWN)) {
+		advLog("Decrease Cooldown before " + beforeAbility + ".", 2);
+		triggerAbility(ABILITIES.COOLDOWN);
```

Comment:
I think this should be `disableCooldownIfRelevant`, as it checks if cooldown is already active in lane.